### PR TITLE
Rename the exported `security-group-inputs` file to `security-group-variables`

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Rename the exported `security-group-inputs.tf` file to `security-group-variables.tf`

## why
* Standardize on file names with variables for specific resources. 
For example, we have `spotinst-variables.tf` for Spot, `security-group-variables.tf` for SG, etc.

### CODE REVIEW

File: exports/security-group-variables.tf [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# security-group-inputs Version: 2
+# security-group-variables Version: 2
```
Comment (nitrocode) [inline exports/security-group-variables.tf:1]: should this version be bumped and use semver ?

**Review by nitrocode — APPROVED** [review]

File: exports/security-group-variables.tf (hunk 1 above)
Comment (aknysh) [reply exports/security-group-variables.tf:1]: prob not related to the file name, but to the file content, which did not change

**Pull request merged**